hivemq-edge/src/main/java/com/hivemq/api/utils/BridgeUtils.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2019-present HiveMQ GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hivemq.api.utils;
+
+import com.hivemq.bridge.config.BridgeTls;
+import com.hivemq.bridge.config.BridgeWebsocketConfig;
+import com.hivemq.bridge.config.CustomUserProperty;
+import com.hivemq.bridge.config.LocalSubscription;
+import com.hivemq.bridge.config.MqttBridge;
+import com.hivemq.bridge.config.RemoteSubscription;
+import com.hivemq.edge.api.model.Bridge;
+import com.hivemq.edge.api.model.BridgeCustomUserProperty;
+import com.hivemq.edge.api.model.BridgeSubscription;
+import com.hivemq.edge.api.model.LocalBridgeSubscription;
+import com.hivemq.edge.api.model.Status;
+import com.hivemq.edge.api.model.WebsocketConfiguration;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.stream.Collectors;
+
+public class BridgeUtils {
+
+    public static @NotNull Bridge convert(final @NotNull MqttBridge mqttBridge, final @NotNull Status status) {
+        return new Bridge().id(mqttBridge.getId())
+                .host(mqttBridge.getHost())
+                .port(mqttBridge.getPort())
+                .clientId(mqttBridge.getClientId())
+                .keepAlive(mqttBridge.getKeepAlive())
+                .sessionExpiry(mqttBridge.getSessionExpiry())
+                .cleanStart(mqttBridge.isCleanStart())
+                .username(mqttBridge.getUsername())
+                .password(mqttBridge.getPassword())
+                .loopPreventionEnabled(mqttBridge.isLoopPreventionEnabled())
+                .loopPreventionHopCount(mqttBridge.getLoopPreventionHopCount() < 1 ?
```

Comment:
Is this a config error or do we expect negative values?

Reply:
We expect 0 as unlimited, i guess negative numbers work the same logically.